tiledb/api/src/filter/strategy.rs

```diff
@@ -37,8 +37,98 @@ fn prop_bitwidthreduction() -> impl Strategy<Value = FilterData> {
     ]
 }
 
-fn prop_compression_reinterpret_datatype() -> impl Strategy<Value = Datatype> {
-    prop_datatype_implemented()
+fn prop_compression_delta_strategies(
+    input_datatype: Option<Datatype>,
+) -> Vec<BoxedStrategy<CompressionType>> {
+    /*
+    let dt_filter = if let Some(input_datatype) = input_datatype {
+        Box::<FnType>::new(|dt: Datatype| {
+            if input_datatype.is_real_type() {
+                dt != Datatype::Any
+            } else {
+                !dt.is_real_type()
+            }
+        })
+    } else {
+        Box::<FnType>::new(|dt: Datatype| !dt.is_real_type())
+    };
+    */
+
+    /*
+    let delta = prop_datatype()
+        .prop_filter(
+            "Input to delta filter cannot be floating-point type",
+            move |dt| dt_filter(*dt),
+        )
+        .prop_map(|dt| CompressionType::Delta {
+            reinterpret_datatype: Some(dt),
+        });
+
+    let double_delta = prop_datatype()
+        .prop_filter(
+            "Input to delta filter cannot be floating-point type",
+            move |dt| dt_filter(*dt),
+        )
+        .prop_map(|dt| CompressionType::DoubleDelta {
+            reinterpret_datatype: Some(dt),
+        });
+
+    vec![delta.boxed(), double_delta.boxed()]
+    */
```

Comment:
Are you needing these? Commented out code makes my eye twitch.

Reply:
Oh oops, I am also twitching!